neutone_sdk/sqw.py

```diff
@@ -45,16 +46,15 @@ def __init__(
         self.channel_normalizer = ChannelNormalizerSandwich(
             use_debug_mode=use_debug_mode
         )
-        # TODO(cm): switch to a more robust resampling method that prevents aliasing
-        self.resample_sandwich = InplaceInterpolationResampler(
+        self.resample_sandwich = Inplace4pHermiteResampler(
             self.in_n_ch,
             self.out_n_ch,
             daw_sr,
             daw_sr,  # Tmp sample rate values
             daw_bs,
             use_debug_mode=use_debug_mode,
         )
-        self.params_resample_sandwich = InplaceInterpolationResampler(
+        self.params_resample_sandwich = InplaceLinearResampler(
```

Comment:
Is there some reason to keep this linear? Because it's faster and we don't care about accuracy in params as much?

Reply:
Yeah the max frequency of the parameters is so much lower than audio (<100 Hz) that the signal is sufficiently oversampled such that a linear interpolator is actually close to optimal given the performance tradeoff.